docs/entityoperator.md

```diff
@@ -0,0 +1,214 @@
+
+### Authorization (ACLs)
+
+- In the above example, the user my-app-user has Read & Write access to my-topic.
+
+Clients will only be able to perform allowed operations.
+
+## How Clients Retrieve and Use Credentials
+
+After creating a KafkaUser, Strimzi automatically generates a Kubernetes Secret with the credentials.
+
+**Example**
+
+```bash
+kubectl get secret my-app-user -o yaml
+```
+
+It will contain
+
+#### For TLS authentication
+
+- ca.crt (CA certificate)
+- user.crt (Client certificate)
+- user.key (Client private key)
+
+#### For SCRAM-SHA authentication
+
+- password (Base64-encoded password)
+
+### Using These Credentials in a Kafka Client
+
+**Example**
+
+##### Java Producer Example (TLS Authentication)
+
+
+```java
+Properties props = new Properties();
+props.put("bootstrap.servers", "my-cluster-kafka-bootstrap:9093");
+props.put("security.protocol", "SSL");
+props.put("ssl.truststore.location", "/etc/secrets/ca.p12");
+props.put("ssl.truststore.password", "password");
+props.put("ssl.keystore.location", "/etc/secrets/user.p12");
+props.put("ssl.keystore.password", "password");
+
+KafkaProducer<String, String> producer = new KafkaProducer<>(props);
+```
+
+#### Java Consumer Example (SCRAM-SHA Authentication)
+
+```java
+Properties props = new Properties();
+props.put("bootstrap.servers", "my-cluster-kafka-bootstrap:9093");
+props.put("security.protocol", "SASL_SSL");
+props.put("sasl.mechanism", "SCRAM-SHA-512");
+props.put("sasl.jaas.config", "org.apache.kafka.common.security.scram.ScramLoginModule required username='my-app-user' password='my-secret-password';");
+
+KafkaConsumer<String, String> consumer = new KafkaConsumer<>(props);
+
+```
+
+### Summary of How Applications Use KafkaTopic & KafkaUser CRs
+
+| Action    | Operator Responsible |
+| -------- | ------- |
+| Developer creates a `KafkaTopic` CR  | Topic Operator creates & syncs the topic in Kafka    |
+| Developer creates a KafkaUser CR | User Operator creates the user & credentials  |
+| Application retrieves credentials from Kubernetes Secrets	| Application mounts the secrets for authentication |
+| Application connects to Kafka using these credentials | Producer/Consumer communicates with Kafka  |
+
+
+## Impact of Central Cluster Failure on Kafka Clients in a Stretch Cluster
+
+In stretch Kafka deployment, where
+
+✅ Kafka brokers and controllers are spread across multiple Kubernetes clusters.<br>
+✅ The central cluster hosts all Kafka CRs, including Kafka, KafkaNodePool, KafkaUser, and KafkaTopic.<br>
+✅ The Entity Operator (managing users & topics) runs in the central cluster.
+
+
+## What About Entity Operator Functions?
+The Entity Operator becomes unavailable when the central cluster goes down. However, this does not impact existing Kafka clients directly because
+
+- Kafka clients do not interact with the Entity Operator at runtime.
+- User authentication still works as long as secrets (TLS/SCRAM) were distributed to all clusters.
```

Comment:
Can you explain why we nee to add this `as long as secrets (TLS/SCRAM) were distributed to all clusters.` 

I would expect Kafka to store user credentials in ZK / controllers and not rely on topic operator implementation.